### DISCUSSION TITLE
Weekly Calendar

### DISCUSSION THREAD

**Discussion by cahitberkay** [fomantic/Fomantic-UI#1665] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
It can be useful if the week option is added in the calendar. It will be especially useful for reporting.

**Comment by lubber-de** [comment]
Do you mean to show week numbers? This is already implemented. Use `showWeekNumbers: true`
See https://jsfiddle.net/lubber/nzues9bk/

![image](https://user-images.githubusercontent.com/18379884/92308080-7596ae00-ef9b-11ea-8105-c0766644a9c7.png)

**Comment by cahitberkay** [comment]
Thank you in the answer but no, what I really needed was that when a day was chosen, he actually chose the week. For example, when he chooses September 8, he chooses the 37th week completely .. In this way, we can bring the report of that week.

**Comment by ko2in** [comment]
Are you talking about to have a built-in method to get week number? Something like:
`$('.ui.calendar').calendar('get weekNumber', argument)`

**Comment by cahitberkay** [comment]
Example; **September 7-13, 2020**

Screenshot;
![Screenshot_2020-09-06 Fomantic-UI Sandbox - JSFiddle - Code Playground](https://user-images.githubusercontent.com/24186944/92330580-a7317700-f078-11ea-90d7-4a891f56c436.png)

**Comment by ko2in** [comment]
You can use the formatter to include the week number for that. See doc [here](https://fomantic-ui.com/modules/calendar.html#custom-format).

**Comment by lubber-de** [comment, marked as answer]
As @ko2in pointed out, use a custom date formatter for your use case
I prepared that for you so far here https://jsfiddle.net/lubber/nzues9bk/46/
![image](https://user-images.githubusercontent.com/18379884/92334425-fb932180-f08d-11ea-8949-2035d6ec9b59.png)
![image](https://user-images.githubusercontent.com/18379884/92334430-0948a700-f08e-11ea-9702-fddf01b4cf0a.png)
